src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -1973,6 +1973,65 @@ export const _handleCreateFormField: ControllerHandler<
   )
 }
 
+/**
+ * NOTE: Exported for testing.
+ * Private handler for POST /forms/:formId/fieldslist
+ * @precondition Must be preceded by request validation
+ * @security session
+ *
+ * @returns 200 with created form fields
+ * @returns 403 when current user does not have permissions to create a form field
+ * @returns 404 when form cannot be found
+ * @returns 409 when form field update conflicts with database state
+ * @returns 410 when creating form field for an archived form
+ * @returns 413 when creating form field causes form to be too large to be saved in the database
+ * @returns 422 when an invalid form field creation is attempted on the form
+ * @returns 422 when user in session cannot be retrieved from the database
+ * @returns 500 when database error occurs
+ * todo: add validation
+ */
+export const _handleCreateFormFields: ControllerHandler<
```

Comment:
This doesn't seem like it has to be preceded by an underscore. Underscore usually implies that the function is used internally by the module, and in particular doesn't really have any value when seen outside of it's surrounding scope (or is part of a larger flow that makes more sense when viewed together). 

I see maybe we intended to implement some kind of validation prior to calling this. it doesn't seem like we have this yet. So, maybe we should remove the underscore for now. Separately, we should delete the comment on validation unless it still needs to be done.